### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by clonker**

### CODE REVIEW

File: deeptime/decomposition/_score.py [hunk 1]
```diff
@@ -288,6 +288,9 @@ def vamp_score_cv(fit_fetch: Union[Estimator, Callable], trajs, lagtime=None, n=
     scores = np.empty((n,), float)
     sliding = splitting_mode == 'sliding'
 
+    if random_state is None or isinstance(random_state, int):
+        random_state = np.random.RandomState(random_state)
```
Comment (thempel) [inline deeptime/decomposition/_score.py:292]: I guess the `else` is an instance of `np.random.RandomState`? Maybe safer to raise if it's not?

File: deeptime/util/data.py [hunk 2]
```diff
@@ -139,12 +149,17 @@ class TimeLaggedDataset:
     ----------
     data : (T, n) ndarray
```
Comment (thempel) [inline deeptime/util/data.py:150]: Should have same dimension as `data_lagged`, (T, m)?

Comment (clonker) [reply deeptime/util/data.py:150]: not necessarily

File: deeptime/decomposition/_score.py (hunk 1 above)
Comment (clonker) [reply deeptime/decomposition/_score.py:292]: yes!

**Pull request merged**